packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -8,6 +8,8 @@ export {
   AxiosDataFetcherConfig,
   NativeDataFetcher,
   NativeDataFetcherConfig,
+  GraphQLRequestClient,
+  GraphQLRequestClientFactoryConfig,
```

Comment:
Think we need to export `GraphQLRequestClientFactory` as well in case they wanted to create their own (and not use our static one).